docs/stencil-docs/developing-further/google-analytics-enhanced-ecommerce.md

```diff
@@ -30,26 +30,25 @@ Cornerstone versions 2.6.0+ will have Data Tags already included in the theme.
 </div>
 </div>
 
-## Prerequisites
+### Prerequisites

```

Comment:
Prerequisites are being listed before mentioning that this is a tutorial. There needs to be some forewarning about this. The prerequisite section should come after the "Adding data tags" heading.

In addition, since this heading is going to remain h3, it should be removed from the "On this page" section.